### DISCUSSION TITLE
Reverse proxy localhost:3333 or 4200?

### DISCUSSION THREAD

**Discussion by charleslcso** [ghostfolio/ghostfolio#1110] [Q&A] [answered]
Hello again,

I followed the Developer guide and finished the installation. 

curl -X POST http://localhost:3333/api/v1/import

and it throws

{"statusCode":401,"message":"Unauthorized"}

Seems to work fine.

Now I want to reverse proxy it using Nginx so that I have access to it from the Internet. Should I proxy 4200 or 3333? Anyone got any guide on how to do this please?

**Comment by dtslvr** [comment, marked as answer]
Port `4200` is the default port of _Angular_ when the application is served locally, so this is the wrong one.

I am using something like this for `nginx.conf`:

```
server {
    listen 443 ssl;
    server_name ghostfol.io;

    location / {
        proxy_pass  http://ghostfolio:3333;
    }
}
```

**Comment by charleslcso** [comment]
Really appreciate your reply!

Thank you!

**Reply by charleslcso** [reply]
Hello,

I just want 1 Nginx on another server to reverse proxy to :3333. As the instance only listens to localhost, how can I change it to listen to 0.0.0.0:3333 or *:3333?

**Reply by dtslvr** [reply]
You could set the environment variable `HOST` (see [here](https://github.com/ghostfolio/ghostfolio#supported-environment-variables)).

**Reply by charleslcso** [reply]
Nice!

Glad to know about this: BASE_CURRENCY.

Should I open a new thread? 

A response of type JSON is replied when I hit http://<ip>:3333. The content is ENOENT: no such file or directory, stat /ghostfolio/dist/apps/client/index.html doesn't exist.

Indeed, /ghostfolio/dist/apps/client does not exist.

In /ghostfolio/dist/apps there is only "api"

**Reply by dtslvr** [reply]
> A response of type JSON is replied when I hit http://:3333. The content is ENOENT: no such file or directory, stat /ghostfolio/dist/apps/client/index.html doesn't exist.
> 
> Indeed, /ghostfolio/dist/apps/client does not exist.
> 
> In /ghostfolio/dist/apps there is only "api"

I' m sorry, I cannot follow what you are trying to achieve.

**Comment by charleslcso** [comment]
I have this, and I believe the installation is not complete. 

![Screen Shot 2022-07-31 at 8 53 57 AM](https://user-images.githubusercontent.com/520537/182005118-051d374e-aff3-47a2-adde-26a2555b81e7.png)

![Screen Shot 2022-07-31 at 8 56 38 AM](https://user-images.githubusercontent.com/520537/182005138-8a9c201c-2071-461f-bf89-060e52915808.png)

If you would give me some pointers that would be nice.

**Reply by charleslcso** [reply]
I installed in Ubuntu on Intel. It is working now.

**Reply by charleslcso** [reply]
Didn't realise my issue is related to this: https://github.com/ghostfolio/ghostfolio/issues/1146
